### DISCUSSION TITLE
Paperspace drawing problem

### DISCUSSION THREAD

**Discussion by melihpeker-pointr** [mozman/ezdxf#462] [Q&A]
Hi, I have a DXF file which contains 3 floors in it. Those floors are drawn together in Model layout and also there are 3 paperspace sheets in which all of those are shown one by one.  When I view that dxf with Autodesk Viewer, everything look fine but when I use cad_viewer.py and the Model layout shows very small, (it shows the layout from very very far away, I need to zoom like 10x for a reasonable view like follows: (there is tiny dot in the center which is the Model layout :) )
![ezdxf](https://user-images.githubusercontent.com/83574547/121482187-90d49500-c9d5-11eb-8f4a-e56c8fd5a5d2.png)
 
Also, when I try to view layouts, although I can view that layout on Autodesk Viewer without a problem, I can see nothing but a viewport entity as following:

![Screenshot 2021-06-10 at 10 22 35](https://user-images.githubusercontent.com/83574547/121482552-ed37b480-c9d5-11eb-9f12-1ab16093b8ee.png)

The question is how to fix the small view on Model layout (I tried to use bbox but it didn't work also) and how to properly draw paperspace layouts which have only viewport entities? 

Thanks a lot!

**Comment by mbway** [comment]
the most likely cause of the cropping issue is that there are entities very far from the main drawing that you want to view. you either need to set the view limits manually or remove the extra entities, either manually or using ezdxf before drawing. If the problem is something else and you provide a minimal example of the problem then a better solution might be possible.

rendering entities through viewports is not implemented so drawing them as rectangles is to be expected. This was not a feature that I needed so I neglected to implement it but it should be possible to do.

**Reply by mozman** [reply]
@mbway How about disabling Paperspace rendering at all (remove"Select Layout" menu)?

**Reply by mbway** [reply]
well, that's up to you if you think it's confusing for people. I would suggest leaving an option like `--show-paperspace` to enable the 'select layout' menu if you did that.

The paper space can sometimes contain useful information (that cannot be found in the modelspace) and not just viewports. For example in floor plan documents, the paper space often contains a sidebar with details such as the address, map name, floor number, date of creation, company who drew the map, department the map belongs to, a small site map to put the map into context etc.
